Formula/proxysql.rb

```diff
@@ -0,0 +1,37 @@
+class Proxysql < Formula
+  desc "High Performance Advanced Proxy for MySQL"
+  homepage "http://www.proxysql.com/"
+  url "https://github.com/sysown/proxysql/archive/v1.4.10.tar.gz"
+  sha256 "29020cba75b778ef45b06c5469ee5930ce80afbd0290a112d8c2a6652b4c8a2e"
+
+  # Build dependencies listed here: https://github.com/sysown/proxysql/blob/master/INSTALL.md
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+  depends_on "make" => :build
+  depends_on "mysql@5.6" => :test
+  depends_on "openssl"
+
+  def install
+    system "make"
+    # Currently, the Proxysql build system uses fixed paths for install locations.
+    # See https://github.com/sysown/proxysql/blob/master/Makefile
+    # The 'make install' target would need to be patched to support Homebrew.
+    # Configurable target locations are requested in https://github.com/sysown/proxysql/issues/1565
+    bin.install "src/proxysql"
+    etc.install "etc/proxysql.cnf"
+  end
+
```

Comment:
Wouldn't you want to add a plist method so that it can be run as a service ?

Reply:
👍